source/plugin/economy/using.rst

```diff
@@ -34,18 +29,24 @@ Example: Loading the EconomyService
 
 .. code-block:: java
 
-	import org.spongepowered.api.event.Listener;
-	import org.spongepowered.api.event.service.ChangeServiceProviderEvent;
 	import org.spongepowered.api.service.economy.EconomyService;
+	import org.spongepowered.api.Sponge;
 
-	private EconomyService economyService;
-
-	@Listener
-	public void onChangeServiceProvider(ChangeServiceProviderEvent event) {
-		if (event.getService().equals(EconomyService.class)) {
-			economyService = (EconomyService) event.getNewProviderRegistration().getProvider();
-		}
+	Optional<EconomyService> serviceOpt = Sponge.getServiceManager().provide(EconomyService.class);
+	if (!serviceOpt.isPresent()) {
+	    // handle there not being an economy implementation
```

Comment:
Basically, any text inside a code block gets ignored by Crowdin, so our translators never get to translate comments inside them. Hence [docs guideline 3.1](https://docs.spongepowered.org/stable/en/contributing/spongedocs.html) was born. That said, if there's no sensible alternative, then leave it where it is.